posts/2020/04/pytorch-tensoriterator-internals.md

```diff
@@ -10,6 +10,12 @@
 .. type: text
 -->
 
+The history section of this post is still relevant, but `TensorIterator`'s
+interface has changed significantly. For an update on the new API, please check
+out [this new blog
+post](../../2021/04/pytorch-tensoriterator-internals-update.html).
```

Comment:
This link doesn't seem to work. I need to find out the correct syntax.

Reply:
Maybe `/blog/2021/04/pytorch-tensoriterator-internals-update.html`

Reply:
Also, [nikola](https://www.getnikola.com/handbook.html#doc) suggests you can do 
```
:doc:`pytorch-tensoriterator-internals-update.`
```